tests/Support/SessionIdValidatorTest.php

```
@@ -0,0 +1,104 @@
+        self::assertSame('valid_session', SessionIdValidator::sanitize('  valid_session  '));
+        self::assertSame('session', SessionIdValidator::sanitize("\tsession\n"));
+        self::assertSame('session', SessionIdValidator::sanitize('session'));
+    }
+
+    public function testConstants(): void
+    {
+        self::assertSame('/^[a-zA-Z0-9_-]+$/', SessionIdValidator::VALID_PATTERN);
+        self::assertSame(256, SessionIdValidator::MAX_LENGTH);
+        self::assertSame(16, SessionIdValidator::MIN_RECOMMENDED_LENGTH);
+    }
+}
```

PHPMD (2.15.0)

The class SessionIdValidatorTest has 13 public methods. Consider refactoring SessionIdValidatorTest to keep number of public methods under 10. (undefined)

(TooManyPublicMethods)